kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -3056,13 +3056,13 @@ def _normalize_ha_network_groups(self, network_groups):
 
             normalized_rules = sorted(normalized_rules, key=lambda r: (r['resource'], r['weight']))
             try:
-                min_available = int(group_cfg.get('minAvailableCount', 1))
+                min_required_score = int(group_cfg.get('minAvailableCount', 1))
             except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)